### ISSUE TITLE
unable to get sleep summary data using time_series('sleep')  

### ISSUE THREAD

**Issue by sehokim88** [orcasgit/python-fitbit#147]
When I make a API request using
   ```
   auth_client.time_series('sleep', base_date=start_date, end_date=end_date)
   ```

, I do not get sleep summary data that contains
   ```
   'summary': {'stages': {'deep': 84, 'light': 263, 'rem': 104, 'wake': 59},
   'totalMinutesAsleep': 463,
   'totalSleepRecords': 1,
   'totalTimeInBed': 510}}
   ```

I was able to get the full sleep data including summary data using just auth_client.sleep() but not using auth_client.time_series('sleep')

**Comment by salvador-moreno** [comment]
I do not think that is an issue, the 'time_series' method is indicated to retrieve only the sleep data in a finer detail,. If you want to get the summary AND the time series you can always use the client's 'sleep' method.

**Comment by jakemiller13** [comment]
![Tracking Sleep Habits Using Your Fitbit](https://user-images.githubusercontent.com/34395972/67143561-95fe3280-f23a-11e9-904e-fdf32bf915f9.jpg)

I'm working on a project that does exactly what you are looking for. It returns the summary as part of a json object, so you just need to pull it out of that.
